src/pages/Account/SettingsTelegramNotifications.js

```diff
@@ -17,15 +17,21 @@ const SettingsTelegramNotifications = ({ classes = {}, description }) => {
     <div className={cx(classes.container, styles.settingBlock)}>
       <Label className={cx(classes.left, 'row v-center')}>
         <span className='mrg--r mrg-xs'>Telegram notifications</span>
-        <AlertTooltip
-          isVisible={!alertNotifyTelegram}
-          content={
-            <span>
-              <span className='txt-m'>Telegram notifications are disabled!</span> This means you
-              will not receive Telegram notifications when this alerts is triggered.
-            </span>
-          }
-        />
+        {!!count && count > 0 && (
```

Comment:
`!!count` is `true` for every number except `0`, hence just `count > 0 && ...` should work fine.